config/migrate/migrate.go

```diff
@@ -145,21 +150,25 @@ func oneToTwoConfig(cfg *config.Config) error {
 		if apiCfg.Address == "" {
 			apiCfg.Address = defaultAPICfg.Address
 		}
-		if apiCfg.WebsocketAddress == "" {
-			apiCfg.WebsocketAddress = defaultAPICfg.WebsocketAddress
-		}
+		// TODO(b5): need a strategy for setting config now that this field is removed
```

Comment:
Think we should drop the code and just create and issue for the next config migration to work out how we maintain older migrations intact if future migrations make a "breaking change".